src/EA.Iws.Web/Areas/NotificationMovements/Controllers/ReceiptRecoveryBulkUploadController.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace EA.Iws.Web.Areas.NotificationMovements.Controllers
+{
+    using System;
+    using System.Web.Mvc;
+    using Infrastructure;
+    using Infrastructure.BulkUpload;
+    using Prsd.Core.Mediator;
+    using ViewModels.PrenotificationBulkUpload;
+
+    [Authorize]
+    public class ReceiptRecoveryBulkUploadController : Controller
+    {
+        private readonly IMediator mediator;
+        private readonly IBulkMovementValidator validator;
+        private readonly IFileReader fileReader;
+
+        public ReceiptRecoveryBulkUploadController(IMediator mediator, IBulkMovementValidator validator, IFileReader fileReader)
+        {
+            this.mediator = mediator;
+            this.validator = validator;
+            this.fileReader = fileReader;
+        }
+
+        [HttpGet]
+        public ActionResult Index(Guid notificationId)
```

Comment:
Can you please add a unit test for this Controller and add a test just for this Index action?

Reply:
Done. The unit test for the post can be added when the data upload page has been implemented.